### DISCUSSION TITLE
Creating restful api with FastAPI

### DISCUSSION THREAD

**Discussion by Ahtii** [fastapi/fastapi#7200] [Questions]
Hi,
I am working as an intern in a local company where we have been assigned a project of creating a realtime chat web application. I am working on the backend where I have to develop a restful api using FastAPI. 

I have been working on this for past few weeks and I have created a register, login and logout endpoint with MySQL database. Today we had a meeting and I showed my work to our CTO of the company and he told me that this is not a restful api. I asked why it is not and he said that you have to find on your own. Can anyone please tell me how it is not a restful api ?

**Comment by raphaelauv** [comment]
FastAPI is a tool to build easily an API  thank to a backend web server

the API will be restFULL or not depending of how you implement it 

https://en.wikipedia.org/wiki/Representational_state_transfer#Architectural_constraints

you can CLOSE this issue , thank you.

**Comment by raphaelauv** [comment]
@Ahtii could you please close the issue , thanks

**Comment by tiangolo** [comment]
Thanks for the help here @raphaelauv ! 👏  🙇 

Sorry, I can't help you if you don't follow the issue template and provide more information.

If you still have problems please create a new issue and follow the template step by step.

Given that, I'm gonna close this one.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
